llz4.luau

```diff
@@ -192,16 +193,15 @@ end
 	  the cost of compression efficiency.
 	@return string The compressed data as a string.
 ]]
-local function compressString(str, acceleration)
+local function compressString(str: string, acceleration: number?): string
 	checkType(str, "string", 1, "compressString")
 	acceleration = checkAcceleration(acceleration, 2, "compressString")
 
 	local buf, len = compressBuffer(buffer.fromstring(str), 0, #str, acceleration)
 	return buffer.readstring(buf, 0, len)
 end
 
-
-local function expandBuffer(out, maxLen)
+local function expandBuffer(out: buffer, maxLen: number)
```

Comment:
Annotating the return type as `(buffer?, any)` seems to appease the linter and is probably good enough